src/tests/unit.rs

```diff
@@ -0,0 +1,70 @@
+use crate::tests::comp;
+
+#[test]
+fn unit_hello() {
```

Comment:
Maybe this file can get the same effect of https://github.com/Ph0enixKM/Amber/pull/238?

Of course once the other one get merged.
I mean move the bash output in static files so it is more easy to update them as I guess that we will update the bash generated because compiler optimization.